public/js/main.js

```diff
@@ -101,3 +85,42 @@ function renderToolbox() {
     document.querySelector("#mount")
   );
 }
+
+if(process.env.NODE_ENV !== "DEVTOOLS_PANEL") {
+  connectClient(tabs => {
+    actions.newTabs(tabs);
+
+    // if there's a pre-selected tab, connect to it and load the sources.
+    // otherwise, just show the toolbox.
+    if (hasSelectedTab()) {
+      const selectedTab = getSelectedTab(store.getState().tabs.get("tabs"));
+      const tab = selectedTab.get("firefox") || selectedTab.get("chrome");
+
+      actions.selectTab({ tabActor: tab.actor });
+      connectThread(tab).then(() => {
+        initPage(actions);
+        renderToolbox();
+      });
+    } else {
+      renderToolbox();
+    }
+  });
+
+  if (isEnabled("chrome.debug")) {
+    chromeTabs(response => {
+      actions.newTabs(response);
+    });
+  }
+}
+else {
+  actions.newTabs([{ actor: "tab1" }]);
+  actions.selectTab({ tabActor: "tab1" });
```

Comment:
Yeah. The problem is that we've integrated the toolbox so we have to have a tab selected, so we just fake one. We don't actually use the tab from the store anywhere, so it's not a big deal. There might be a better way to do it; I'll think about it some more.